packages/server/jiraImagesHandler.ts

```diff
@@ -31,9 +29,12 @@ const getImageFromCache = async (
 let jiraPlaceholderBuffer: Buffer | undefined
 const servePlaceholderImage = async (res: HttpResponse) => {
   if (!jiraPlaceholderBuffer) {
-    jiraPlaceholderBuffer = await fsp.readFile(
-      path.join(__dirname, jiraPlaceholder.slice(__webpack_public_path__.length))
-    )
+    try {
+      const res = await fetch(jiraPlaceholder)
```

Comment:
all other assets need a public URL except this one, so instead of special casing this one I just turned it into a fetch vs. a file system read.